### ISSUE TITLE
Initial value null

### ISSUE THREAD

**Issue by nbickel** [JohannesHoppe/angular-date-value-accessor#3]
My init value was null and writeValue failed.

i wrote
    if(value != null)
      this._renderer.setElementProperty(this._elementRef.nativeElement, 'valueAsDate', value);

**Comment by JohannesHoppe** [comment]
Good point. Im not sure about the exact behviour. I would clear the value instead.
I will fix that in a timely manner

**Comment by JohannesHoppe** [comment]
(which means 1-2 weeks, since I need to finish a super-urgent project first)

**Comment by iwes** [comment]
I ended up using the following, as the dates being passed in were not true `Date` objects:

```
    writeValue(value: Date): void {
        if (value != null) {
            this._renderer.setElementProperty(this._elementRef.nativeElement, 'valueAsDate', new Date(value));
        }
    }
```

If it becomes necessary for my project, I'll do more work to check that the input value resolves to a valid `Date` before attempting to set it.

**Comment by JohannesHoppe** [comment]
closed by #5

**Comment by JohannesHoppe** [comment]
Thanks!
v0.0.2 is available on NPM. Please report any errors, if required.
